examples/05-fsm.janet

```diff
@@ -1,26 +1,27 @@
-(use ./../junk-drawer)
+(use /junk-drawer)
 
 # Example 1
 #
 # this macro is used to define the state "factory"
 # The goto method is used to transition between states.
-# the Enter and Leave methods are optional, and are called during the GOTO method.
+# the Enter and Leave methods are optional, and are called during the
+# GOTO method.
 (fsm/define
- colored-warnings
- (green
-  :enter (fn [self] (print "entering green"))
-  :leave (fn [self] (print "leaving green"))
-  :warn  (fn [self]
-           (print "before warn")
-           (:goto self :yellow)
-           (print "after warn")))
- (yellow
-  :enter (fn [self] (print "entering yellow"))
-  :panic |(:goto $ :red)
-  :clear |(:goto $ :green))
- (red
-  :leave (fn [self] (print "leaving red"))
-  :calm  |(:goto $ :yellow)))
+  colored-warnings
+  {:green
+   {:enter (fn [self] (print "entering green"))
+    :leave (fn [self] (print "leaving green"))
+    :warn (fn [self]
+            (print "before warn")
+            (:goto self :yellow)
+            (print "after warn") :)}
```

Comment:
is the `:` a typo?

Reply:
Oh, yes, That is the typo. I will fix it. Kakoune multiselections I blame for this.